cmd/openshift-tuned.go

```diff
@@ -194,6 +201,10 @@ func getConfig() (*rest.Config, error) {
 	return nil, fmt.Errorf("Could not locate a kubeconfig")
 }
 
+func getJitter(period int64, factor float64) int64 {
+	return rand.Int63n(int64(float64(period)*factor+1)) - int64(float64(period)*factor/2)
```

Comment:
Why between 4320 to 8641?

Reply:
I'm afraid I'm not sure where the numbers you mention come from.  Perhaps -4320 and 4320 when `resyncPeriodPodDefault` is being used to call this function, i.e. `resyncPeriodPodDefault + getJitter(resyncPeriodPodDefault, 0.3)`? This function on its own gives you a "jitter" with a factor around a certain period.  If the period is 100 and jitter 0.1, you'd get jitter -5--5 and a random period from 95-105 as a result.  I just happened to choose a larger period and jitter of 0.3.  Is this the question?  Why the jitter of 0.3?

Reply:
Yes I see, that was calculated with the 2/3 calls `getJitter(resyncPeriodPodDefault, 0.3)`
Missed the third call with a different period.